doc/v2/dev/index_en.rst

```diff
@@ -1,9 +1,27 @@
 Development
 ------------
 
+
+PaddlePaddle adheres to the following three sections of code and document specifications.
+
+
+PaddlePaddle uses git for version management and docker is used for building and testing environment. The code includes Cuda, C++, Python, Shell and other programming languages，which comply with Google C++ Style, Pep-8, and the code base includes style checking by an automatic inspection tool. Code comments need to follow the Doxygen specification, and if code that does not meet the style requirements will fail to compile with. we provide the following guidelines for the use of git, build tests and code development.
+..  toctree::
+  :maxdepth: 1
+
+  contribute_to_paddle_cn.md
+
+
+PaddlePaddle is suitable for domestic and foreign users, including Chinese and English documents. We recommend the English version of design documents and problem descriptions. The design documents focus on problem descriptions, backgrounds, and followed by solutions. As documents are generated by Sphinx, code comments should comply with the Sphinx document standard. We recommend to use the paddlepaddle.org tool to compile and generate and preview documents locally. Please refer to:
```

Comment:
I do not suggest to write the first sentence. The sentence makes PaddlePaddle sound like a localized project. We should instead write something like this:

_PaddlePaddle is well documented in English and Chinese. We recommend using the English version of the documents and problem description._

Reply:
The design documents focus on problem descriptions, backgrounds, and followed by solutions. -> The design documents focus on problem descriptions, backgrounds and are followed by solutions.

Reply:
Sphinx document standard -> Sphinx **documentation** standard